src/nyc_trees/apps/login/forms.py

```diff
@@ -24,7 +25,7 @@ class ForgotUsernameForm(forms.Form):
 class NycRegistrationForm(RegistrationFormUniqueEmail):
     tos = forms.BooleanField(
         widget=forms.CheckboxInput,
-        label='I agree to the terms of use',
+        label=mark_safe('I agree to the <a href="http://www1.nyc.gov/home/terms-of-use.page" target="new">terms of use</a>'),  # NOQA
```

Comment:
No need to turn off linting. Python auto joins strings when they are separated only by whitespace.
```python
        label=mark_safe('I agree to the <a href="'
                        'http://www1.nyc.gov/home/terms-of-use.page'
                        '" target="new">terms of use</a>')
```